interfaces/builtin/io_ports_control.go

```diff
@@ -20,34 +20,105 @@
 package builtin
 
 import (
+	"bytes"
+	"fmt"
+
 	"github.com/snapcore/snapd/interfaces"
 )
 
-const ioPortsControlConnectedPlugAppArmor = `
+var ioPortsControlConnectedPlugAppArmor = []byte(`
```

Comment:
Can you change this back to const? While we don't use this consistently within the codebase (for older interfaces), this is the coding practice for new interfaces.